src/main/java/uk/gov/companieshouse/chipsrestinterfacesconsumer/client/ChipsRestClient.java

```diff
@@ -0,0 +1,38 @@
+package uk.gov.companieshouse.chipsrestinterfacesconsumer.client;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+import uk.gov.companieshouse.chipsrestinterfacesconsumer.model.ChipsKafkaMessage;
+
+import javax.annotation.PostConstruct;
+import java.util.Collections;
+
+@Service
+public class ChipsRestClient {
+
+    private static final String CHIPS_REST_ENDPOINT_URI_VAR = "{chips-rest-endpoint}";
+
+    @Value("${CHIPS_REST_INTERFACES_HOST}")
```

Comment:
this is probably one for a new story, but it would be better if the app refused to start if the host was missing from config. I think Jo suggested we do this when we configured the email sending via kafka in extensions (and was probably done the same way in our other apps). She suggested reading the configs in via a spring config class/bean and that should fail the app starting if it couldn't find values. Not sure if it auto fails or whether it is a manual thing, I'd have to check the configs for email.

Reply:
Otherwise we'd get messages ending up in error topic. Although this doesn't solve the problem if we put an incorrect host name - i guess we can't account for that.

Reply:
If CHIPS_REST_INTERFACES_HOST spring will fail to construct this class causing the app to fallover on startup
(though if the config is set to an invalid url, the app will start, and only fall over when it tries to send to the url)

Reply:
ah ok, thought it might just populate with null